modules/acquia_cms_common/tests/src/Kernel/AcquiaCmsTelemetryTest.php

```diff
@@ -29,8 +50,14 @@ protected function setUp(): void {
       $this->container->get("extension.list.module"),
       $this->container->get("http_client"),
       $this->container->get("config.factory"),
-      $this->container->get("state")
+      $this->container->get("state"),
+      $this->container->getParameter("site.path"),
     );
+    $this->configFactory = $this->container->get("config.factory");
+    $this->sitePath = $this->container->getParameter('site.path');
```

Comment:
@rajeshreeputra can we do like this?
$path = explode('/', $this->container->getParameter('site.path'));
$this->sitePath = end($path);

Reply:
Sure, we can.